package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@appannie/react-infinite-calendar",
-  "version": "3.3.5",
+  "version": "3.3.6",
```

Comment:
Let's remove the version bump since that'll mess with the pkg release process.

Reply:
got it